tcp_driver/include/tcp_driver/http_client.hpp

```diff
@@ -0,0 +1,322 @@
+// Copyright 2022 MAP IV, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef TCP_DRIVER__HTTP_CLIENT_HPP_
+#define TCP_DRIVER__HTTP_CLIENT_HPP_
+
+#include <array>
+#include <string>
+#include <vector>
+
+#include <boost/beast.hpp>
```

Comment:
How is Beast installed? I saw that it's a header-only library, so it should be added as a vendor package

Reply:
Beast has been included in Boost since version [1.66](https://www.boost.org/users/history/version_1_66_0.html). There is no need to manually install it since [Ubuntu 20.04](https://packages.ubuntu.com/search?suite=focal&arch=any&searchon=all&keywords=boost)

Reply:
Thanks for the explanation